packages/ibm-products-web-components/src/globals/internal/icon-loader-utils.ts

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
oh hmm these files should be able to be imported from `@carbon/web-components`...although it is in the `internal` folder

`import { iconLoader } from "@carbon/web-components/es/globals/internal/icon-loader.js";`

Reply:
Ohhh haha! I didn't know those are included in the build. I'll make some changes to fix that.